Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/SwitchRowView.kt

```diff
@@ -0,0 +1,43 @@
+package de.rki.coronawarnapp.ui.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.TextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.google.android.material.switchmaterial.SwitchMaterial
+import de.rki.coronawarnapp.R
+
+class SwitchRowView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : ConstraintLayout(context, attrs, defStyleAttr) {
+
+    private val switch by lazy { findViewById<SwitchMaterial>(R.id.switch_view) }
+    private val title by lazy { findViewById<TextView>(R.id.switch_title) }
+    private val subtitleView by lazy { findViewById<TextView>(R.id.switch_subtitle) }
+
+    init {
+        LayoutInflater.from(context).inflate(R.layout.switch_row, this, true)
+    }
+
+    fun setChecked(turnedOn: Boolean?) {
```

Comment:
Why `Boolean?` and not `Boolean` ?

Reply:
For data binding. Can be null initially.